### ISSUE TITLE
[BUG] System ModifyPlayerStats > Player is not loaded to the cache

### ISSUE THREAD

**Issue by crashzk** [K4ryuu/K4-System#119]
I'm getting an error log:
```
2024-01-24 23:06:31.439 -03:00 [INFO] plugin:K4-System Initializing 'ModuleRank'
2024-01-24 23:06:31.708 -03:00 [INFO] plugin:K4-System Checking table 'k4ranks' | Validating 6 columns
2024-01-24 23:06:32.134 -03:00 [INFO] plugin:K4-System Initializing 'ModuleStat'
2024-01-24 23:06:32.210 -03:00 [INFO] plugin:K4-System Checking table 'k4stats' | Validating 19 columns
2024-01-24 23:06:32.298 -03:00 [INFO] plugin:K4-System Initializing 'ModuleTime'
2024-01-24 23:06:32.375 -03:00 [INFO] plugin:K4-System Checking table 'k4times' | Validating 10 columns
2024-01-24 23:06:32.457 -03:00 [INFO] plugin:K4-System Initializing 'ModuleUtils'

2024-01-25 14:43:22.296 -03:00 [WARN] plugin:K4-System ModifyPlayerStats > Player is not loaded to the cache (HRP    F4KE    awp)
2024-01-25 14:43:22.296 -03:00 [WARN] plugin:K4-System ModifyPlayerStats > Player is not loaded to the cache (HRP Lzkiwnl  愛)
2024-01-25 14:43:22.419 -03:00 [WARN] plugin:K4-System ModifyPlayerStats > Player is not loaded to the cache (⑳ hrp nezK)
2024-01-25 14:43:22.420 -03:00 [WARN] plugin:K4-System ModifyPlayerPoints > Player is not loaded to the cache (⑳ hrp nezK)
2024-01-25 14:43:22.425 -03:00 [WARN] plugin:K4-System ModifyPlayerStats > Player is not loaded to the cache (dma)
2024-01-25 14:43:22.425 -03:00 [WARN] plugin:K4-System ModifyPlayerStats > Player is not loaded to the cache (c a y k y)
2024-01-25 14:43:22.425 -03:00 [WARN] plugin:K4-System ModifyPlayerStats > Player is not loaded to the cache (junincs)
2024-01-25 14:43:22.425 -03:00 [WARN] plugin:K4-System ModifyPlayerStats > Player is not loaded to the cache (𝐙𝐢𝐤𝐚sᴄᴄᴘ)
2024-01-25 14:43:22.426 -03:00 [WARN] plugin:K4-System ModifyPlayerStats > Player is not loaded to the cache (d-)
```
[log-K4-System20240124.txt](https://github.com/K4ryuu/K4-System/files/14056285/log-K4-System20240124.txt)
[log-K4-System20240125.txt](https://github.com/K4ryuu/K4-System/files/14056286/log-K4-System20240125.txt)

**Plugin Version:** "K4-System" (3.1.2 (release)) by K4ryuu

**Comment by K4ryuu** [comment]
Greetings,
This after map changes randomly?

**Comment by crashzk** [comment]
> Greetings, This after map changes randomly?

I completely turned off the server, deleted all logs and turned it on again.

I didn't even connect to it and checked the logs and there are these:
```
2024-01-25 15:12:12.156 -03:00 [INFO] plugin:K4-System Initializing 'ModuleRank'
2024-01-25 15:12:12.424 -03:00 [INFO] plugin:K4-System Checking table 'k4ranks' | Validating 6 columns
2024-01-25 15:12:12.834 -03:00 [INFO] plugin:K4-System Initializing 'ModuleStat'
2024-01-25 15:12:12.907 -03:00 [INFO] plugin:K4-System Checking table 'k4stats' | Validating 19 columns
2024-01-25 15:12:12.993 -03:00 [INFO] plugin:K4-System Initializing 'ModuleTime'
2024-01-25 15:12:13.066 -03:00 [INFO] plugin:K4-System Checking table 'k4times' | Validating 10 columns
2024-01-25 15:12:13.147 -03:00 [INFO] plugin:K4-System Initializing 'ModuleUtils'
```
After changing the map, new logs were generated:
```
2024-01-25 15:29:35.700 -03:00 [WARN] plugin:K4-System ModifyPlayerStats > Player is not loaded to the cache (Ncr ( CS.PRO ))
2024-01-25 15:30:21.281 -03:00 [WARN] plugin:K4-System ModifyPlayerStats > Player is not loaded to the cache (Ncr ( CS.PRO ))
2024-01-25 15:30:53.126 -03:00 [WARN] plugin:K4-System ModifyPlayerStats > Player is not loaded to the cache (Ncr ( CS.PRO ))
2024-01-25 15:31:29.098 -03:00 [WARN] plugin:K4-System ModifyPlayerStats > Player is not loaded to the cache (Ncr ( CS.PRO ))
2024-01-25 15:31:32.131 -03:00 [WARN] plugin:K4-System ModifyPlayerStats > Player is not loaded to the cache (Ncr ( CS.PRO ))
2024-01-25 15:31:46.581 -03:00 [WARN] plugin:K4-System ModifyPlayerStats > Player is not loaded to the cache (KӨΣ)
2024-01-25 15:31:46.581 -03:00 [WARN] plugin:K4-System ModifyPlayerStats > Player is not loaded to the cache (crashzk)
2024-01-25 15:31:46.582 -03:00 [WARN] plugin:K4-System ModifyPlayerStats > Player is not loaded to the cache (yure1)
2024-01-25 15:31:46.582 -03:00 [WARN] plugin:K4-System ModifyPlayerStats > Player is not loaded to the cache (愛と憎しみ)
2024-01-25 15:31:46.582 -03:00 [WARN] plugin:K4-System ModifyPlayerStats > Player is not loaded to the cache (♕GIL♕)
2024-01-25 15:31:46.582 -03:00 [WARN] plugin:K4-System ModifyPlayerStats > Player is not loaded to the cache (Ncr ( CS.PRO ))
2024-01-25 15:31:46.719 -03:00 [WARN] plugin:K4-System ModifyPlayerStats > Player is not loaded to the cache (KӨΣ)
2024-01-25 15:31:46.719 -03:00 [WARN] plugin:K4-System ModifyPlayerStats > Player is not loaded to the cache (crashzk)
2024-01-25 15:31:46.719 -03:00 [WARN] plugin:K4-System ModifyPlayerStats > Player is not loaded to the cache (yure1)
2024-01-25 15:31:46.719 -03:00 [WARN] plugin:K4-System ModifyPlayerStats > Player is not loaded to the cache (愛と憎しみ)
2024-01-25 15:31:46.719 -03:00 [WARN] plugin:K4-System ModifyPlayerStats > Player is not loaded to the cache (♕GIL♕)
2024-01-25 15:31:46.719 -03:00 [WARN] plugin:K4-System ModifyPlayerStats > Player is not loaded to the cache (Ncr ( CS.PRO ))
```

**Comment by K4ryuu** [comment]
Thanks, 
I'm checking it now!

**Comment by K4ryuu** [comment]
The problem has been solved in V3.1.3. 
Appreciate your help
